### PR TITLE
Browser cache I18n fix

### PR DESCRIPTION
**Pull request by jiteshjha**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [x] `development.sqlite.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jiteshjha** [comment]
Hello @jywarren ! I have added the last 4 digits of Time.now to the redirected URL with a simple test. Any suggestions?

File: app/controllers/settings_controller.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@ def change_locale
     lang = I18n.default_locale unless I18n.available_locales.include?(lang)
     cookies.permanent[:plots2_locale] = lang
     I18n.locale = lang
-    redirect_to request.referer || root_url
+    if request.referer
+      redirect_to request.referer+ "?_=" + Time.now.to_i.to_s[-4..-1]
```
Comment (jywarren) [inline app/controllers/settings_controller.rb]: can you add an extra space before the `+`,  like `request.referer+ "?_=" + `... ? Very small thing, I know, but it's very helpful to have exact coding practices over the long term. Thank you !

Comment (jywarren) [inline app/controllers/settings_controller.rb]: Was there a problem with simply `Time.now.to_i.to_s` ? It's used elsewhere in the codebase and the extra integers aren't a big deal. I think perhaps consistency is preferable here.

File: test/integration/I18n_test.rb [hunk 2]
```diff
@@ -176,4 +176,24 @@ class I18nTest < ActionDispatch::IntegrationTest
...
+      # Set the language as Deutsch
+      lang=:de
+      get_via_redirect "/change_locale/"+lang.to_s
+      assert_equal "#{lang}", "#{I18n.locale}"
+      
+      # Move back to English(Default locale)
+      lang=:en
+      get_via_redirect "/change_locale/"+lang.to_s
+      assert_equal "#{lang}", "#{I18n.locale}"
+      assert true
```
Comment (jywarren) [inline test/integration/I18n_test.rb]: `assert true` is an example assertion which will never fail, so why don't we remove it here and from line 176 too? Thank you!

**Comment by jywarren** [comment]
Hi, excellent; i left a couple small requests and can merge once they're fixed. Thank you very much!

**Comment by jiteshjha** [comment]
Hello, @jywarren ! I've added the requested changes, and re-based branch with the latest master.
Thanks!

**Comment by jywarren** [comment]
Manually merged. Thanks!

**Pull request closed without merging**